Analysis/ScanLibraries/source/XiaListModeDataDecoder.cpp

```diff
@@ -227,7 +227,8 @@ std::pair<unsigned int, unsigned int> XiaListModeDataDecoder::DecodeWordZero(con
 
     data.SetChannelNumber(word & mask.GetChannelNumberMask().first);
     data.SetSlotNumber((word & mask.GetSlotIdMask().first) >> mask.GetSlotIdMask().second);
-    data.SetCrateNumber((word & mask.GetCrateIdMask().first) >> mask.GetCrateIdMask().second);
+    // Crate number in Pixie list-mode data is ignored
+    data.SetCrateNumber(0);
```

Comment:
We need to ask nori why he does/wants this? The crate number should be in the list data, if it's not that's a bug in the firmware. If it is why doesn't he want to track it?